monero-outreach-docs/translations/nl/PSA/files/forks.md

```diff
@@ -0,0 +1,29 @@
+# Ken je Hard Forks
```

Comment:
# Ken de veiligheid van beurzen en wisselkantoren

Beurzen kunnen handig zijn om Monero te (ver)kopen, maar er zijn onderlinge verschillen.

## Onthou het volgende om je XMR veilig te houden bij het gebruik van een beurs:

1. Doe je eigen research. Probeer de volgende vragen te beantwoorden:

- a. Zijn er positieve reviews van onafhankelijke bronnen? Zijn ze betrouwbaar?

- b. Is de beurs ooit aangevallen? Was de reactie daarop gepast?

- c. Hoe lang is hij al operationeel?

- d. Zijn de oprichters/managers bekend? Hebben deze individuen of entiteiten een goede reputatie?

- e. Kan je je privacy behouden? Hoef je geen persoonlijke informatie te delen?

- f. Hanteert de website beveiligingsmaatregelen zoals "captcha" en "2FA" (Twee-Factor-Authenticatie)? Zijn de beveilingscerificaten legitiem en start het adres met "https://"?

- g. Indien je belastingaangifte wil doen: kan je bewijsstukken van je transacties zonder moeite downloaden? 

2. Beurzen creëren een (eigen) wallet in jouw plaats. Waarschijnlijk heb je geen toegang tot de hersteltekst. Als de beurs wordt aangevallen of bedrog blijkt te zijn, verlies je alles in deze wallet. Een goede vuistregel is er nooit langdurig XMR te bewaren: indien je het niet op zeer korte termijn spendeert, breng het dan over naar een wallet die je zelf beheert.

3. Vooraleer je een significant bedrag overbrengt naar een beurs dien je best een testtransactie te plaatsen. Gebruik een testbedrag dat je kan veroorloven te verliezen.

4. Kijk altijd het zend- en ontvangstadres na bij elke XMR-transactie. Doe dit zelfs als je kopieert en plakt, want er bestaan aanvallen waarbij je crypto-adres wordt veranderd. Moest zulke malware je apparaat hebben besmet, voldoet simpel nakijken wat je plakt om je te beschermen.

5. Op termijn zullen gedecentraliseerde beurzen gemeengoed worden. Deze liggen qua ethos in the verlengde van de gedecentraliseerde cryptovaluta Monero. Overweeg ze te gebruiken, maar doe je research!

Reply:
# Ken je hard forks

Voor non-programmeurs kan de term "hard fork" verwarrende, zelfs negatieve connotaties dragen. Misschien heb je een paar artikels gelezen over "hard forks" en hoe dramatisch ze verdeling hebben gezaaid onder de cryptoprojecten. De "hard forks" van Monero nemen interessante vormen aan, en vallen ruwweg in drie categorieën:

### 1) Netwerkupgrade / regelmatig geplande hark fork

Monero-programmeurs streven naar twee hard forks per jaar. Deze worden geleid door Monero's kernprogrammeurs, steeds volgens gemeenschapsconsensus. Typisch betreft het verscheidene verbeteringen aan de kerncode. Het is belangrijk dat je altijd je Monero-wallet(s) upgrade wanneer dit gebeurt, de verantwoorlijkheid valt op niemands schouders dan de jouwe. [verder gepreek weglaten?]

Anderen hebben gebruikgemaakt van Monero's hard forks door hun projecten te starten op de oude keten. Deze projecten falen vaak op korte termijn, gezien de vaak hebzuchtige motivatie. Als je je wallet niet update voor een hard fork maar je toch XMR zendt, verlies je ze mogelijk op een andere keten, of verlies je tijd ze te recuperen.

_Veiligheidsniveau:_ upgrade je Monerowallet zodra er updates beschikbaar zijn.

### 2) Hard forks van nieuwe, legitieme projecten


Sommige projecten splitsen af van Monero, en gebruiken de onderliggende code voor hun eigen doel. Verschillende factoren bepalen hun legimiteit. 

- geen pre-mine of insta-mine van munten voor de programmeurs
- je privésleutel delen is geen vereiste
- geen controverse over het project of al te negatieve, alarmerend recensies van de Monero-gemeenschap
- het project is open-bron en dus "geeft terug" aan Monero


_Veiligheidsniveau:_ Als het project legitiem is, hoef je niets te doen met je Monero-wallet. Gebruik je gezond verstand en ga je gang.

### 3) Hard fork van oplichters en aanvallers

Deze projecten hebben de omgekeerde kenmerken van de legitieme projecten. Ze worden gedreven door hebzucht, zowel van de kant van de programmeurs als van de gebruikers (lees: slachtoffers). In sommige gevallen, vooral als het delen van je privésleutel vereist is, zijn het bedrieglijke pogingen het Moneronetwerk en de privacy/anonimiteit van haar gebruikers aan te vallen.

_Veiligheidsniveau:_ Sterk aangeraden deze munten te vermijden. Ga door op eigen risico, en begrijp dat deelname ook onbedoeld het Moneronetwerk kan verzwakken.